### DISCUSSION TITLE
Can i get some guide to use cacheControl  with ApolloServer ?

### DISCUSSION THREAD

**Discussion by JANDA-Developers** [MichalLytek/type-graphql#919] [Q&A]
While reading the Apollo server documentation, I found out that Cache is available.
[https://www.apollographql.com/docs/apollo-server/performance/caching/](https://www.apollographql.com/docs/apollo-server/performance/caching/)
I'm using type-graphql, so I need something in the form of a decorator.
I'm not sure how to link these two libraries.

Can I get an example or hint?

**Comment by MichalLytek** [comment]
> Can I get an example or hint?

Of course!
All you need is to use search instead of asking questions 😉 
https://github.com/MichalLytek/type-graphql/tree/master/examples/apollo-cache

**Reply by JANDA-Developers** [reply]
Thanks!!!

**Reply by StarpTech** [reply]
The example is no longer valid for Apollo 3.
